R/logging-tools.R

```diff
@@ -257,3 +257,7 @@ log_browser_version <- function(data_storage, input) {
     )
   })
 }
+
+parse_val <- function(val) {
+  jsonlite::fromJSON(ifelse(is.null(val), "\"\"", val))
+}
```

Comment:
We may remove this later in a separate PR, but keeping it for now because `log_input()` uses it.